app/algorithms/__init__.py

```diff
@@ -1,10 +1,12 @@
 from algorithms.olc import olc_suffix, olc_dynamic, olc
-from algorithms.scs import scs
+from algorithms.scs import scs as scs
+from algorithms.scs import greedy_scs as greedy_scs
```

Comment:
masło maślane... `as` służy do nadawania innych nazw jak masz na przykład:
```from algorithm.scs import do_assembly
from algorithm.olc import do_assembly
```
to używa się `as` żeby się nie nadpisywały. Takie nazwy jak są, są ok.